src/components/table/table.tsx

```diff
@@ -734,6 +734,7 @@ export class Table {
             >
                 <limel-checkbox
                     onChange={this.selectAllOnChange}
+                    disabled={!this.totalRows ? true : false}
```

Comment:
You can use double exclamation point here. That will convert a truthy or falsy value (here that is totalRows) to “true” or “false”. 
```suggestion
                    disabled={!!this.totalRows}
```

Reply:
Thats true 🙂

Reply:
thank you! ❤Only, we need to have disabled in case, when there is no row, so ` disabled={!this.totalRows} `

Reply:
> Only, we need to have disabled in case, when there is no row, so `disabled={!this.totalRows}`

Isn't that what I wrote in the suggested change? (I'm asking because I _think_ you missed my suggested change, but it's Monday and I'm tired, so it's completely possible that I'm staring at some difference that just isn't registering in my brain… 😅)

Reply:
oh👀😃 now I see your comment @adrianschmidt . I guess my thread wasn't updated/ or delayed, while I added my comment. At that time I saw only Malin comment, but it's great that we had the same solution 😉